Its.Log.Monitoring/TelemetryExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Its.Log.Monitoring
+{
+    public static class TelemetryExtensions
+    {
+        public static StatefulResult<IEnumerable<T>, long> CountOf<T>(this IEnumerable<T> sequence, Func<T, bool> selector)
```

Comment:
None of these are actually extending telemetry. They may be related to assertions. The first one looks more related to `Enumerable`.